set-mongo/src/main/java/eu/europeana/set/mongo/service/PersistentUserSetService.java

```diff
@@ -119,5 +119,7 @@ public interface PersistentUserSetService extends AbstractNoSqlService<Persisten
 	 */
 	List<String> getDuplicateUserSetsIds(UserSet userSet);
 
+	long[] getEntityGalleriesMetrics();
```

Comment:
please create 2 methods, one method/metrics and give expresive names to the methods